### DISCUSSION TITLE
Add examples for omitting rounding and/or precision on the Math placeholder

### DISCUSSION THREAD

**Discussion by Starmism** [PlaceholderAPI/PlaceholderAPI#673] [(2) Support]
### Confirmation

- [X] I checked the Issues and Pull request tab for any existing issues or PRs.
- [X] My Feature Request is for **PlaceholderAPI** and not any expansion or other plugin.

### Type

New API feature

### Description

On the list of placeholders, the math expansion only has examples for specifying neither of the two options or specifying both. Whereas a much more common use-case involves only specifying precision and omitting rounding, yet there are no examples for that. I see a lot of questions about the formatting of the math expansion and I honestly myself did not know for sure until Blitz told me and showed me an example. 

Also this definitely does not fall under New API Feature, but there were no categories for wiki improvements sadly :(.

**Comment by Andre601** [comment]
> - [x] My Feature Request is for **PlaceholderAPI** and not any expansion or other plugin.

This is not a PAPI issue.
Also, the wilki shows `[precision]:[rounding]` which are indicators for optional placeholders.
You're the only one so far that had issues with this, so not that much of an issue here.

**Reply by broken1arrow** [reply]
That example get rely confusing, specially when they has remove rounding: 5 to something like this math_5 . 

I has to check my old math placeholders to se how i has set up rounding, it has be more confusing an the old method (i know he probably have to change that to get it some new added thing to work). 

I will say is not only math but other has made it pretty unclear how to build the placeholders (I know is hard to explain it so all understand).

**Comment by Starmism** [comment]
Sadly, there is no form entry for a wiki issue.
Also, the wiki does not show explicit syntax examples for each case, or even one, and I have seen a lot of people with issues related to that.

**Comment by Andre601** [comment]
People should then imo experiment around or ask the devs of the expansions about the quirks of the expansion.

But it should be fairly common knowledge that `<>` indicates required and `[]` indicates optional features.
If you really want examples added, could you just [PR it yourself](https://github.com/PlaceholderAPI/PlaceholderAPI/tree/docs/wiki/wiki#readme).

**Comment by iGabyTM** [comment]
I personally use `[]` for required and `()` for optional args, is not a standard format imo.

**Comment by Starmism** [comment]
And plus it doesn't specify behavior with the `:`. Apparently it is omitted, since Blitz showed me `%math_0_1+1%` is correct, though that makes me wonder what having only the rounding would look like.

**Comment by Andre601** [comment]
You would then check the source/main repo of the expansion: https://github.com/Andre601/Math-expansion#math_1_56477

Again: Why should I now update only this part? It should be either all or nothing imo and we can't explain everything to them.

**Comment by Starmism** [comment]
That's not even the syntax Blitz showed me. And the common scenarios for people are: none, only precision, and precision + rounding. I say you have all three listed for clear visibility.
